src/javascripts/components/cards/sectorCards.js

```diff
@@ -0,0 +1,25 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const sectorCard = (object) => {
+  let buttons = '';
+  const user = firebase.auth().currentUser;
+  if (user) {
+    buttons += `<a href='#' id='${object.firebaseKey}' class='btn btn-primary edit-sectors'>Edit</a>
+      <a id='delete__${object.firebaseKey}' href='#' class='delete btn btn-danger'>Delete</a>
+  `;
+  }
+
+  const domString = $('#app').append(`
+    <div class="weapons" id="sectors-${object.firebaseKey}">
```

Comment:
Why are you using class weapons for the sectors?

Reply:
That's copypasta. You are right it should be removed.